### PR TITLE
(feat) Update start visit and visit notes forms to the new tablet design

### PR DESCRIPTION
**Pull request by Jexsie**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary

- Updates the tablet view of the start visit and visit notes forms to match the new tablet look 
- Checkout the **[start visit](https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/60d5c9eff1d57d142406f38c)** and the **[visit note](https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/60d5e3a96011db1220bd41f4)** designs

## Screencasts
[Screencast from 2023-06-28 10-53-56.webm](https://github.com/openmrs/openmrs-esm-patient-chart/assets/104269786/a5900669-c948-403c-9a8a-1bb143f0addc)
[Screencast from 2023-06-28 10-57-37.webm](https://github.com/openmrs/openmrs-esm-patient-chart/assets/104269786/865b7722-236f-429b-a1fe-ec8bea6554ff)

## Other

- Ignore the font changes. Had issues with importmaps so fonts were not loaded.

### CODE REVIEW

File: packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx [hunk 1]
```diff
@@ -385,7 +385,9 @@ const VisitNotesForm: React.FC<DefaultWorkspaceProps> = ({ closeWorkspace, patie
           </Column>
         </Row>
         <Row className={styles.row}>
-          <Column sm={1}></Column>
+          <Column sm={1}>
+            <span className={styles.columnLabel}>{t('secondaryDiagnosis', 'Secondary Diagnosis')}</span>
```
Comment (Jexsie) [inline packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx:389]: Updated titles in the Diagnosis section to match the [design](https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/60d5e3a96011db1220bd41f4)

File: packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx [hunk 2]
```diff
@@ -356,76 +361,90 @@ const StartVisitForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, closeWor
...
-              <BaseVisitType
-                onChange={(visitType) => {
-                  setVisitType(visitType);
-                  setIsMissingVisitType(false);
-                }}
-                visitTypes={allVisitTypes}
-                patientUuid={patientUuid}
-              />
-            )}
+            <div className={styles['sectionField']}>
```
Comment (Jexsie) [inline packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx:364]: Only one simple change here. I wrapped everything under a section field

File: packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx [hunk 3]
```diff
@@ -277,7 +277,7 @@ const VisitNotesForm: React.FC<DefaultWorkspaceProps> = ({ closeWorkspace, patie
         </Row>
         <Row className={styles.row}>
           <Column sm={1}>
-            <span className={styles.columnLabel}>{t('diagnosis', 'Diagnosis')}</span>
+            <span className={styles.columnLabel}>{t('primaryDiagnosis', 'Primary Diagnosis')}</span>
```
Comment (denniskigen) [inline packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx:280]:
```suggestion
            <span className={styles.columnLabel}>{t('primaryDiagnosis', 'Primary diagnosis')}</span>
```

File: packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx (hunk 1 above)
Comment (denniskigen) [inline packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx:389]:
```suggestion
            <span className={styles.columnLabel}>{t('secondaryDiagnosis', 'Secondary diagnosis')}</span>
```

File: packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx [hunk 4]
```diff
@@ -356,76 +361,90 @@ const StartVisitForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, closeWor
...
+              />
+            </div>
           </section>
 
           {/* Queue location and queue fields. These get shown when queue location and queue fields are configured */}
-          {config.showServiceQueueFields && <ExtensionSlot name="add-queue-entry-slot" />}
+          {config.showServiceQueueFields && (
+            <section>
+              <div className={styles.sectionTitle}></div>
+              <div className={styles['sectionField']}>
```
Comment (denniskigen) [inline packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx:443]:
```suggestion
              <div className={styles.sectionField}>
```

**Review by denniskigen — COMMENTED** [review]
Generally LGTM. I've left a few tiny stylistic suggestions.

File: packages/esm-patient-chart-app/src/visit/visit-form/visit-form.scss [hunk 5]
```diff
@@ -81,3 +81,23 @@
...
+      min-width: 8rem;
+      text-align: left;
+    }
+
+    .sectionField {
+      flex-basis: 70%;
+    }
+  }
+ }
+}
```
Comment (denniskigen) [inline packages/esm-patient-chart-app/src/visit/visit-form/visit-form.scss:102]:
```suggestion
```suggestion
:global(.omrs-breakpoint-lt-desktop) {
  .container {
    & section {
      display: flex;
      gap: 4rem;

      .sectionTitle {
        flex-basis: 30%;
        min-width: 8rem;
        text-align: left;
      }

      .sectionField {
        flex-basis: 70%;
      }
    }
  }
}
```
```

File: packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx [hunk 6]
```diff
@@ -356,76 +361,90 @@ const StartVisitForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, closeWor
...
 
           {/* Visit type attribute fields. These get shown when visit attribute types are configured */}
           <section>
-            <VisitAttributeTypeFields
-              setVisitAttributes={setVisitAttributes}
-              isMissingRequiredAttributes={isMissingRequiredAttributes}
-              visitAttributes={visitAttributes}
-              setErrorFetchingResources={setErrorFetchingResources}
-            />
+            <div className={styles.sectionTitle}>{isTablet && t('visitAttributes', 'Visit Attributes')}</div>
```
Comment (denniskigen) [inline packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx:428]:
```suggestion
            <div className={styles.sectionTitle}>{isTablet && t('visitAttributes', 'Visit attributes')}</div>
```

File: packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx [hunk 7]
```diff
@@ -319,7 +319,12 @@ const StartVisitForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, closeWor
 
           {/* Upcoming appointments. This get shown when upcoming appointments are configured */}
           {config.showUpcomingAppointments && (
-            <ExtensionSlot state={upcomingAppointmentState} name="upcoming-appointment-slot" />
+            <section>
+              <div className={styles.sectionTitle}></div>
```
[the remark this answers is not in the document]
Comment (Jexsie) [reply packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx:323]: this is a title section divider. It helps in flexing

[the remark this answers is not in the document]
Comment (denniskigen) [reply packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx:323]: Ok. Thanks for clarifying.

File: packages/esm-patient-chart-app/translations/en.json [hunk 8]
```diff
@@ -135,6 +135,7 @@
   "unknown": "Unknown",
   "updateError": "Error updating upcoming appointment",
   "visit": "Visit",
+  "visitAttributes": "Visit Attributes",
```
Comment (denniskigen) [inline packages/esm-patient-chart-app/translations/en.json:138]:
```suggestion
  "visitAttributes": "Visit attributes",
```

**Comment by denniskigen** [comment]
I don't see the accompanying design changes to the visit notes form either from the screencasts or from running your changes locally. Did I miss something, @Jexsie?

**Comment by Jexsie** [comment]
> I don't see the accompanying design changes to the visit notes form either from the screencasts or from running your changes locally. Did I miss something, @Jexsie?


works fine on my side
[Screencast from 2023-06-30 11-50-19.webm](https://github.com/openmrs/openmrs-esm-patient-chart/assets/104269786/1bc4b064-4d4c-4199-bb8a-686ec995f0cd)

**Comment by denniskigen** [comment]
> > I don't see the accompanying design changes to the visit notes form either from the screencasts or from running your changes locally. Did I miss something, @Jexsie?
> 
> works fine on my side [Screencast from 2023-06-30 11-50-19.webm](https://github.com/openmrs/openmrs-esm-patient-chart/assets/104269786/1bc4b064-4d4c-4199-bb8a-686ec995f0cd)

Seen it. Thanks.

**Review by denniskigen — APPROVED** [review]
Thanks, @Jexsie!

**Pull request merged**